docs/source/changelog.md

```diff
@@ -4,6 +4,31 @@
 
 ## [Unreleased]
 
+## 2.0
+
+### 2.0.0 - 2021-11-22
+
+#### Breaking changes
+
+A breaking change was introduced in [#545](https://github.com/jupyterhub/kubespawner/pull/545), making the default value of `allow_privilege_escalation` be `False`. Practically a consequence of this is that a user can't use `sudo` unless this or the `privileged` configuration is explicitly set to `true`. Technically it means that the JupyterHub user Pod that KubeSpawner creates will have a container with a `securityContext` that has `allowPrivilegeEscalation` set to `false` by default now.
```

Comment:
I don't think `privileged` needs to be true - that has a special meaning quite different from root. I think the only difference with this change is that `allow_privilege_escalation` needs to be set to True, not `privileged`

Reply:
```suggestion
A breaking change was introduced in [#545](https://github.com/jupyterhub/kubespawner/pull/545), making the default value of `allow_privilege_escalation` be `False`. Practically a consequence of this is that a user can't use `sudo` unless `allow_privilege_escalation` is explicitly set to `True`. Technically it means that the JupyterHub user Pod that KubeSpawner creates will have a container with a `securityContext` that has `allowPrivilegeEscalation` set to `false` by default now.
```

Ah, I meant to say something relating to...

> AllowPrivilegeEscalation is true always when the container is: 1) run as Privileged OR 2) has CAP_SYS_ADMIN.

But, this is written out below, so why make the text more complicated by including it in this paragraph as well.